documentation/src/main/asciidoc/topics/ref_hotrod_multimapcachemanager.adoc

```diff
@@ -0,0 +1,10 @@
+[id='remotecache_multimap_cachemanager-{context}']
+= Multimap Cache Manager
+The `MultimapCacheManager` interface is the entry point to get a `RemoteMultimapCache`.
+
+Hot Rod clients can perform multimap cache operations as in the following example:
+
+[source,java]
+----
+include::code_examples/MultimapCacheManagerRemote.java[]
+----
```

Comment:
Can we add a link to the Javadocs? I can only see `org.infinispan.multimap.api.embedded.MultimapCacheManager` when I build the Javadocs from the branch. Is there a remote API or is it the same as embedded?

Reply:
This is a remote api but I don't think we need any link to the javadocs. I did like the counters